src/mpi-base.jl

```diff
@@ -300,6 +300,55 @@ function Comm_split_type(comm::Comm,split_type::Integer,key::Integer;info::Info=
     MPI.Comm(newcomm[])
 end
 
+function Dims_create!(nnodes::Integer, ndims::Integer, dims::Vector{Cint})
+    ccall(MPI_DIMS_CREATE, Nothing, (Ref{Cint}, Ref{Cint}, Ptr{Cint}, Ref{Cint}),
+          nnodes, ndims, dims, 0)
+end
+
+function Dims_create!(nnodes::Integer, dims::Array{T,N}) where T <: Integer where N
```

Comment:
It looks like `dims` can be `AbstractArray{T, N}`, because you make a copy into a `Cint` array.

Reply:
OK